ichnaea/log.py

```diff
@@ -25,29 +26,43 @@ def configure_logging():
     logging_level = settings("logging_level")
 
     if local_dev_env:
-        handlers = ["console"]
+        handlers = ["dev"]
+        # Prepare structlog logs for local dev ProcessorFormatter
+        structlog_fmt_prep = structlog.stdlib.ProcessorFormatter.wrap_for_formatter
```

Comment:
The last processor in the chain prepares the log for the handler. ``ProcessorFormatter.wrap_for_formatter`` is specific to ``ProcessorFormatter``, and is a good fit for logs created with ``structlog``.  ``ProcessorFormatter`` does some extra work to handle logs generated from stdlib ``logging`` formatters, but they end up looking similar.